.github/workflows/test.yml

```diff
@@ -1,31 +1,118 @@
-name: Build & Test
+name: Build das api components
+# This workflow uses github runners.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# This may be adjusted to whatever suits best your runners config.
+# Current config will build on manual trigger or pull-request (each push)
 on:
-  push:
```

Comment:
So just to confirm, what this will do is not run the CI when you are pushing to your dev branch, but then as soon as you open a PR to any branch, it will run on the PR branch every time there's a check in.  That seems like the right behavior.

But should we add something like:
```
push:
    branches: [main]
```
so that CI will run in the case that we do a hotfix directly to main without a PR?

Reply:
Ok, if later on more branches are needed,it's simple to expand the list.
If there is any other testing scenario yoou may need, let me know.